.envrc

```
@@ -0,0 +1,13 @@
+if ! find_up ./.venv/checkvenv.txt; then
+    python3 -m venv .venv
+    touch ./.venv/checkvenv.txt
+fi
+source ./.venv/bin/activate
+
+if ! find_up ./.venv/spawnpoint.txt; then
+    pip install .
```

```suggestion
    pip install --editable .
```
So that you can import your code and have edits made available right away.